src/reynir/bindb.py

```diff
@@ -108,16 +108,27 @@ def meanings(self, w: str) -> List[BIN_Tuple]:
         ]
 
     @lru_cache(maxsize=_NAME_GENDER_CACHE_SIZE)
-    def lookup_name_gender(self, name: str) -> str:
+    def lookup_name_gender(self, name: str, preferred_case: str = "nf") -> str:
```

Comment:
Perhaps add * to force keyword-only parameter? i.e. `(self, name: str, *, preferred_case: str = "nf")`